### ISSUE TITLE
CA1027 fires for simple aliases

### ISSUE THREAD

**Issue by tannergooding** [dotnet/roslyn-analyzers#6982]
### Analyzer

**Diagnostic ID**: [CA1027](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1027): `Mark enums with FlagsAttribute`

### Steps To Reproduce

1. Create a new C# Class Library, targeting `net8.0`
2. Edit the csproj and add `<AnalysisLevel>latest-all</AnalysisLevel>`
3. Define the following enum:
```csharp
public enum MyEnum
{
    Value,
    Alias = Value,
}
```

### Expected behavior

No warning should be given as members are not powers of two nor combinations of other values. There is simply an alias introduced, such as might be done to correct a typo or provide a less ambiguous name.

### Actual behavior

The analyzer is triggered.
